### ISSUE TITLE
getting error after install gem with rails 4.2

### ISSUE THREAD

**Issue by Olefine** [glebm/to_spreadsheet#19]
```
Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet/rails/action_pack_renderers.rb:3:in `require': cannot load such file -- action_controller/responder (LoadError)
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet/rails/action_pack_renderers.rb:3:in `<top (required)>'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet/railtie.rb:3:in `require'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet/railtie.rb:3:in `<top (required)>'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet.rb:4:in `require'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet.rb:4:in `<top (required)>'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bundler-1.10.5/lib/bundler/runtime.rb:76:in `require'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bundler-1.10.5/lib/bundler/runtime.rb:76:in `block (2 levels) in require'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bundler-1.10.5/lib/bundler/runtime.rb:72:in `each'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bundler-1.10.5/lib/bundler/runtime.rb:72:in `block in require'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bundler-1.10.5/lib/bundler/runtime.rb:61:in `each'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bundler-1.10.5/lib/bundler/runtime.rb:61:in `require'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/bundler-1.10.5/lib/bundler.rb:134:in `require'
	from /Users/egorgorodov/dev/ruby/goparcel/config/application.rb:7:in `<top (required)>'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/railties-4.1.4/lib/rails/commands/commands_tasks.rb:146:in `require'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/railties-4.1.4/lib/rails/commands/commands_tasks.rb:146:in `require_application_and_environment!'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/railties-4.1.4/lib/rails/commands/commands_tasks.rb:68:in `console'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/railties-4.1.4/lib/rails/commands/commands_tasks.rb:40:in `run_command!'
	from /Users/egorgorodov/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/railties-4.1.4/lib/rails/commands.rb:17:in `<top (required)>'
	from bin/rails:4:in `require'
	from bin/rails:4:in `<main>'
```

Can we fix it?

p.s How i can generate xlsx and put result to file for storing locally?

**Comment by marcalc** [comment]
I confirm this also happens with 3.2.22 and ruby-2.2.2

/Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/dependencies.rb:251:in `require': cannot load such file -- action_controller/responder (LoadError)
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/dependencies.rb:251:in `block in require'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/dependencies.rb:236:in `load_dependency'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/dependencies.rb:251:in `require'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet/rails/action_pack_renderers.rb:3:in `<top (required)>'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/dependencies.rb:251:in `require'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/dependencies.rb:251:in `block in require'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/dependencies.rb:236:in `load_dependency'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/dependencies.rb:251:in `require'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet/railtie.rb:3:in `<top (required)>'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/dependencies.rb:251:in `require'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/dependencies.rb:251:in `block in require'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/dependencies.rb:236:in `load_dependency'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-3.2.22/lib/active_support/dependencies.rb:251:in `require'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/to_spreadsheet-1.0.5/lib/to_spreadsheet.rb:4:in `<top (required)>'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/bundler-1.10.6/lib/bundler/runtime.rb:76:in `require'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/bundler-1.10.6/lib/bundler/runtime.rb:76:in `block (2 levels) in require'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/bundler-1.10.6/lib/bundler/runtime.rb:72:in `each'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/bundler-1.10.6/lib/bundler/runtime.rb:72:in `block in require'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/bundler-1.10.6/lib/bundler/runtime.rb:61:in `each'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/bundler-1.10.6/lib/bundler/runtime.rb:61:in `require'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/bundler-1.10.6/lib/bundler.rb:134:in `require'
	from /Users/marcalc/Code/medscale/medscale-edc/config/application.rb:8:in `<top (required)>'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/railties-3.2.22/lib/rails/commands.rb:53:in `require'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/railties-3.2.22/lib/rails/commands.rb:53:in `block in <top (required)>'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/railties-3.2.22/lib/rails/commands.rb:50:in `tap'
	from /Users/marcalc/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/railties-3.2.22/lib/rails/commands.rb:50:in `<top (required)>'
	from script/rails:6:in `require'
	from script/rails:6:in `<main>'

**Comment by marcalc** [comment]
In this file: to_spreadsheet-1.0.5/lib/to_spreadsheet/rails/action_pack_renderers.rb
I tried changing the require path from ```require 'action_controller/responder'``` to ```require 'action_controller/metal/responder'``` and it worked.

**Comment by Olefine** [comment]
@marcalc @glebm Last comment truth for me also.

**Comment by glebm** [comment]
Please submit a merge request, I will merge it

**Comment by glebm** [comment]
Released in v1.0.6

**Comment by AhmedKamal20** [comment]
@glebm 
It still Happening : 
Ruby : 2.3.1
Rails : 4.2.7.1
to_spreadsheet 1.0.6
```
/home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `require': cannot load such file -- zip/zip (LoadError)
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `block in require'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `require'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/axlsx-1.3.6/lib/axlsx.rb:26:in `<top (required)>'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `require'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `block in require'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `require'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/to_spreadsheet-1.0.6/lib/to_spreadsheet/renderer.rb:1:in `<top (required)>'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `require'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `block in require'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `require'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/to_spreadsheet-1.0.6/lib/to_spreadsheet.rb:3:in `<top (required)>'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/bundler-1.14.6/lib/bundler/runtime.rb:91:in `require'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/bundler-1.14.6/lib/bundler/runtime.rb:91:in `block (2 levels) in require'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/bundler-1.14.6/lib/bundler/runtime.rb:86:in `each'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/bundler-1.14.6/lib/bundler/runtime.rb:86:in `block in require'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/bundler-1.14.6/lib/bundler/runtime.rb:75:in `each'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/bundler-1.14.6/lib/bundler/runtime.rb:75:in `require'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/bundler-1.14.6/lib/bundler.rb:107:in `require'
	from /home/akamal/Damlag/cinematic/config/application.rb:10:in `<top (required)>'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/commands/commands_tasks.rb:78:in `require'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/commands/commands_tasks.rb:78:in `block in server'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/commands/commands_tasks.rb:75:in `tap'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/commands/commands_tasks.rb:75:in `server'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /home/akamal/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/commands.rb:17:in `<top (required)>'
	from bin/rails:4:in `require'
	from bin/rails:4:in `<main>'
```
